language/namespaces.xml

```diff
@@ -59,15 +59,15 @@ version="1.1">
     </listitem>
     <listitem>
      <simpara>
-      Die Möglichkeit, um einen Alias (oder einen Kurznamen) für Sehr_Lange_Namen
-      einzuführen, die angelegt wurden, um das erste Problem zu umgehen. Damit kann
+      Die Möglichkeit, einen Alias (oder einen Kurznamen) für Sehr_Lange_Namen
+      einzuführen, die angelegt wurde, um das erste Problem zu umgehen. Damit kann
```

Comment:
You can do that in your branch, and push, or just commit the following suggestion:
```suggestion
      einzuführen, die geschaffen wurde, um das erste Problem zu umgehen. Damit kann
```

Reply:
Is it now added?

Reply:
Yes, see da25cba32aa2dd9a385bdd258be8ba0995aa4bda.